ui/src/components/Sidebar/SidebarRow.tsx

```diff
@@ -4,12 +4,13 @@ import styles from './styles.module.css';
 interface Props {
   icon: string;
   text: string;
+  onClick?: () => void;
   //   isActive: boolean;
 }
 
 export const SidebarRow: React.FC<Props> = props => {
   return (
-    <div className={styles.container_row}>
+    <div className={styles.container_row} onClick={props.onClick}>
```

Comment:
This will throw an error when onClick is not provided by the parent.

Something like this should make sure that we don't call `onClick` when it's null.

```tsx
<div className={styles.container_row} onClick = {() => props.onClick && props.onClick()}>
```